README.md

```diff
@@ -126,7 +126,7 @@ Use one of [built-in reporters](https://devexpress.github.io/testcafe/documentat
 
 ### Straightforward Continuous Integration
 
-TestCafe is easy to set up on popular Continuous Integration platforms as it allows you to test against various browsers: local, remote or cloud (e.g., [Sauce Labs](https://saucelabs.com/)).
+TestCafe is easy to set up on popular Continuous Integration platforms as it allows you to test against various browsers: local, remote, cloud (e.g., [Sauce Labs](https://saucelabs.com/)) or headless (e.g. [Nightmare](http://www.nightmarejs.org)).
```

Comment:
I mean we already have recipe for saucelabs - link it. For nightmare - just give link to plugin